src/main/java/net/fabricmc/loader/entrypoint/minecraft/EntrypointPatchHook.java

```diff
@@ -391,6 +392,9 @@ public void process(FabricLauncher launcher, Consumer<ClassNode> classEmitter) {
 					finishEntrypoint(type, it);
 				}
 				patched = true;
+			} else if (isRunnable || entrypoint.contains("com.mojang.rubydung.RubyDung")) {
+				// RubyDung should not need any real patches
```

Comment:
Won't it need a patch to add `EntrypointClient#start` (done via `EntrypointPatchHook#finishEntrypoint` right now) so that entrypoints are run?

Reply:
As I said before, it runs without all this.

Reply:
Quite right it runs without it, but you'll also not be getting either `main` or `client` entrypoints running as the call is responsible for both:
https://github.com/FabricMC/fabric-loader/blob/b6f179763dc0924c60dc902cd7735de28fe6a3a9/src/main/java/net/fabricmc/loader/entrypoint/minecraft/hooks/EntrypointClient.java#L26-L34